### DISCUSSION TITLE
Can you add to documentation example for nginx ingress + cert issuer + hetzner load balancer

### DISCUSSION THREAD

**Discussion by stufently** [vitobotta/hetzner-k3s#485] [Q&A]
I can  not full understand how do it with hetzner load balancer 
How right open 80 port for http01 letsencrypt check

**Comment by vitobotta** [comment]
Please use Discussions for anything that is not a bug or an issue with the tool. I'll convert this one so others can help too.

**Comment by vitobotta** [comment]
Could you explain a little more about what you're trying to do? What's the issue you're facing, and what have you tried so far?
